napari/utils/settings/_defaults.py

```diff
@@ -8,6 +8,35 @@
 
 from ..events.evented_model import EventedModel
 from ..notifications import NotificationSeverity
+from ..theme import available_themes
+
+
+class Theme(str):
```

Comment:
I'm fine with this for now... but I don't love it.  It would be the first place where we're using `str` to represent something that is more like an Enum (though I recognize that the dynamic nature of the keys presents a challenge).  I don't have anything better to suggest without playing with it, so it's fine as a first pass.  I don't have strong feelings about where it goes.

Reply:
Ok, maybe let's leave it here for now, and then we can do a pass through this file. It might be more clear what to do as we add more things like it too

Reply:
Well dynamic enums is kinda funny concept :)